cg/services/order_validation_service/models/errors.py

```diff
@@ -71,5 +71,10 @@ class RepeatedSampleNameError(CaseSampleError):
 
 
 class InvalidFatherSexError(CaseSampleError):
-    field: str = "sex"
+    field: str = "father"
     message: str = "Father must be male"
+
+
+class InvalidFatherCaseError(CaseSampleError):
```

Comment:
Slightly unclear naming but probably workable. I would suggest `FatherNotInCaseError`.

Reply:
Nice, changed it https://github.com/Clinical-Genomics/cg/pull/3492/commits/59c55d28cdd64bd4530f77b20ba88e7aa2f3cfda